zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/exceptions/SessionsTokenExpiredException.java

```diff
@@ -1,6 +1,6 @@
-package com.github.onsdigital.zebedee.session.store.exceptions;
+package com.github.onsdigital.zebedee.session.service.exceptions;
```

Comment:
Is this subclasses needed anymore? It doesn't offer anything the parents class doesn't, the only reason I can think of is that we use the exception class to determine specific error types. If that it not used then we could just delete this class.

Reply:
Removed.